gatsby/src/components/Hero/Hero.module.scss

```diff
@@ -0,0 +1,86 @@
+@use '~@nice-digital/nds-core/scss/core';
+
+.hero {
+  @include core.nds-links-inverse;
+  @include core.nds-inverse-button-focus;
+  background: core.$nds-colour-nice-white;
+  color: core.$nds-colour-text;
+  left: 50%;
+  margin-bottom: core.rem(core.$nds-spacing-large);
+  margin-left: -50vw;
+  margin-right: -50vw;
+  overflow: hidden;
+
+  padding-bottom: core.rem(core.$nds-spacing-large);
+  padding-top: core.rem(core.$nds-spacing-large);
+
+  position: relative;
+  right: 50%;
+  width: 100vw;
+
+  a {
+    &:link,
+    &:visited {
+      color: core.$nds-colour-link;
+    }
+
+    &:hover {
+      color: core.$nds-colour-link-hover;
+    }
+
+    &:active {
+      color: core.$nds-colour-text-inverse;
+    }
+  }
+
+  .heroContainer {
+    @include core.nds-container;
+  }
+
+  .title {
+    @include core.nds-h1;
+  }
+
+  .intro {
+    @include core.nds-lead;
+    max-width: unset;
+  }
+
+  .lastUpdated {
+    h2 {
+      @include core.nds-h5;
+      margin: 0;
+    }
+
+    time {
+      @include core.nds-h3;
+      display: block;
+      font-weight: 900;
```

Comment:
We don't have 900 weight any more, see https://github.com/nice-digital/nice-design-system/pull/245

Reply:
Changed to 700 weight, can use the text-shadow hack if this isn't enough